src/Mesh.cc

```diff
@@ -30,6 +30,9 @@ using namespace sdf;
 // Private data class
 class sdf::Mesh::Implementation
 {
+  /// \brief Mesh simplification method
+  public: std::string simplification;
```

Comment:
maybe use a enum here?

Reply:
added enum in [64d7c97](https://github.com/gazebosim/sdformat/pull/1380/commits/64d7c97607c953202b8a771a7c7b3a2e67873774)